qemu/tests/qemu_img.py

```diff
@@ -177,6 +178,45 @@ def create_test(cmd):
                 cluster_size=params.get("image_cluster_size"))
         remove(img)
 
+    def send_signal(timeout=360):
```

Comment:
For this case, the default value is enough. Keep the parameter here is for the subsequent cases we develop.